main.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"text/template"
+
+	"github.com/giantswarm/microerror"
+)
+
+func dupIP(ip net.IP) net.IP {
+	dup := make(net.IP, len(ip))
+	copy(dup, ip)
+	return dup
+}
+
+func main() {
+	err := mainError()
+	if err != nil {
+		panic(fmt.Sprintf("%#v", err))
+	}
+}
+
+func mainError() error {
+	var err error
+
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	bridgeIP := flag.String("bridge-ip", "", "IP address of the bridge (used to retrieve ip and gateway).")
+	hostname := flag.String("hostname", "", "Hostname of the tenant node.")
+	mainConfig := flag.String("main-config", "", "Path to main ignition config (appended to small).")
+	out := flag.String("out", "", "Path to save resulting ignition config.")
+	flag.Parse()
```

Comment:
I would at least do here sanity check that the required values are not empty,  maybe even print them on stdout so in the logs we can see what is used.

Reply:
All the flags are mandatory. So if any of flags values are missed - there will be an error in logs